samples/diagnostics/program-visualization/README.md

```diff
@@ -0,0 +1,22 @@
+# Visualization of Q# Programs #
+
+This sample demonstrates the use of the `%trace` magic command in IQ# to visualize your Q# program within a Jupyter notebook.
```

Comment:
I get this doesn't exist yet, but I'd suggest making a link that's ready for the next release.
```suggestion
This sample demonstrates the use of the [`%trace` magic command](https://docs.microsoft.com/qsharp/api/iqsharp-magic/trace) in IQ# to visualize your Q# program within a Jupyter notebook.
```